framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -1915,25 +1944,6 @@ def is_core_destroy_command(self, command_name):
             return True
         return False
 
-    def generate_structure_type(self, type_name):
-        # Make the type all upper case
-        upper_type = re.sub('([a-z0-9])([A-Z])', r'\1_\2', type_name).upper()
-        type_with_prefix = upper_type
-
-        # Apply any structure type prefix first
-        for api_data in self.valid_khronos_supported_api_data:
-            upper_prefix = api_data.type_prefix.upper()
-            if upper_type.startswith(upper_prefix):
-                type_with_prefix = api_data.struct_type_prefix + upper_type
-
-        type_with_prefix = type_with_prefix.replace('_OPEN_GLES', '_OPENGL_ES_')
-        type_with_prefix = type_with_prefix.replace('_OPEN_GL', '_OPENGL_')
-        type_with_prefix = type_with_prefix.replace('_D3_D11', '_D3D11')
-        type_with_prefix = type_with_prefix.replace('_D3_D12', '_D3D12')
-        type_with_prefix = type_with_prefix.replace('_EGL', '_EGL_')
-        type_with_prefix = type_with_prefix.replace('Device_IDProp', 'Device_ID_Prop')
-        return type_with_prefix
-
```

Comment:
You removed this block, but what fixed it so that it was no longer required?  Is it now OpenXR specific?

Reply:
It looks to me like this was a function to mangle a CamelCase struct name to "screaming snake case" sType.  It looks like sType is now fetched from the XML for the struct.  I think that's handled by the change to the call site for `generate_structure_stype` which now just uses the `values` field, which in the spec looks like it is now always the `sType`.  I'm not sure when this changed but the original code Dustin checked in for this was from 2018 (8a9edb103e), so it's possible it's just that old and the registry has been fixed since then.